File: src/main/java/me/thesourcecode/sourcebot/api/utility/SourceColor.java
package me.thesourcecode.sourcebot.api.utility;

import java.awt.*;

public enum SourceColor {
    BLUE(rgb(52, 152, 219)),
    GREEN(rgb(46, 204, 113)),
    RED(rgb(231, 76, 60)),
    PURPLE(rgb(155, 89, 182)),
    ORANGE(rgb(230, 126, 34));

    private final Color color;

    SourceColor(Color color) {
        this.color = color;
    }

    private static Color rgb(int r, int g, int b) {
        return new Color(r, g, b);
    }

    public Color asColor() {
        return color;
    }
}

File: src/main/java/net/sourcebot/api/misc/SourceColor.kt
package net.sourcebot.api.misc

import java.awt.Color

private fun rgb(r: Int, g: Int, b: Int) = Color(r, g, b)

enum class SourceColor(val color: Color) {
    INFO(rgb(52, 152, 219)),
    SUCCESS(rgb(46, 204, 113)),
    WARNING(rgb(230, 126, 34)),
    ERROR(rgb(231, 76, 60));
}